README.md

```diff
@@ -154,6 +154,8 @@ The current version has not yet reached General Availability version standards.
 In line with our Greptime 2024 Roadmap, we plan to achieve a production-level
 version with the update to v1.0 in August. [[Join Force]](https://github.com/GreptimeTeam/greptimedb/issues/3412)
```

Comment:
1.0 is now planned to be releases at the end of this year?

Reply:
Yes, need to be updated

Reply:
Maybe simply delete this paragraph

Reply:
Updated the description.